release-process.md

```diff
@@ -24,6 +24,26 @@ This document includes:
  - npm publish
  - after remix_live is updated, drop the zip (from the root folder of remix-live repo) to the release.
 
+# remix-ide beta release
+ - git fetch origin master
+ - git checkout origin/master
+ - git checkout -b bumpVersion
+ - update package.json version to the new version "vx.x.x-beta.1"
+ - remove package-lock/json version and generate a new one with `npm install`
+ - merge PR
+ - git fetch origin master
+ - git checkout origin/master
+ - git tag v(version-number) (with "vx.x.x-beta.1")
+ - git push --tags
+ - github-changes -o ethereum -r remix-ide -a --only-pulls --use-commit-body --only-merges --between-tags previous_version...next_version
+ - publish a beta release in github using the changelog
+ - rm -rf node_modules
+ - npm install
+ - remove all soljson.js files in root folder
+ - npm run build
+ - npm publish
+ - drop zip file to the beta release
```

Comment:
from where?

Reply:
https://github.com/ethereum/remix-live-alpha

Reply:
the place where to get the zip file should be put in there